sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 2.0.0b5 (Unreleased)
 
+  **Breaking Changes**
+  - `EventGridEvent` has two additional required positional parameters namely, `data` and `data_version`.
```

Comment:
It's a bug fix, since no calls could succeed in the past without those two, it means people had to add them manually. So this PR is mostly acknowledging to people that yes, our IntelliSense was crap.